release-notes/9.0/supported-os.json

```diff
@@ -0,0 +1,304 @@
+{
+    "channel-version": "9.0",
+    "last-updated": "2024-05-01",
+    "operating-systems": [
+        {
+            "family": "Android",
+            "distributions": [
+                {
+                    "name": "Android",
+                    "lifecycle-policy": "https://support.google.com/android",
+                    "architectures": [
+                        "x64",
+                        "arm64",
+                        "arm"
+                    ],
+                    "supported-cycles": [
+                        "API 21+"
+                    ]
+                }
+            ]
+        },
+        {
+            "family": "Apple",
+            "distributions": [
+                {
+                    "name": "iOS",
+                    "lifecycle-policy": "https://support.apple.com/ios/",
+                    "architectures": [
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "12.2"
+                    ]
+                },
+                {
+                    "name": "iOS Simulator",
+                    "lifecycle-policy": "https://support.apple.com/ios/",
+                    "architectures": [
+                        "x64",
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "12.2",
+                        "13",
+                        "14",
+                        "15",
+                        "16",
+                        "17"
+                    ]
+                },
+                {
+                    "name": "tvOS",
+                    "lifecycle-policy": "https://support.apple.com/apple-tv/",
+                    "architectures": [
+                        "x64",
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "12.2",
+                        "13",
+                        "14",
+                        "15",
+                        "16",
+                        "17"
```

Comment:
I'm a bit confused why we use `X.Y+` for one platform and explicitly list versions for other platforms.

In the current supported-os.md document we only specify the minimum platform version that we support running on.

Reply:
Mainly because of the mixup of min-sdk and supported-cycles so far. supported-cycles will change during the lifetime of a .NET version due to EOL, min-sdk will usually not change anymore and is also only one version not a full list.